### PR TITLE
Module 1: Set up "Hello Microverse" project

### PR DESCRIPTION
**Pull request by ikennarichard**
## Module 1: Set up "Hello Microverse" project
For this project, I had to:
- Create a simple webpage using HTML and CSS.
-  Use Linters and GitHub flow to check for bugs and inconsistencies.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,76 @@
-# Hello-world
\ No newline at end of file
+<a name="user-content-readme-top" id=readme-top></a>
+
+# Hello Microverse
+
+
+# 📗 Table of Contents
```
Comment (topeogunleye) [inline README.md:6]: - [ ] In the case of README consider not leaving any of the sections in the [Microverse README](https://github.com/microverseinc/readme-template) template as it's a project requirement.
  - The only optional section is FAQ.

For Example, you could add an instructions on how to run your projects locally:

## EXAMPLE

```
## Getting Started
To get a local copy up and running, follow the steps in Setup below.

### Prerequisites
Basic knowledge of HTML, CSS, and JavaScript.

### Setup
If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.

[Download the ZIP from this location](https://github.com/ikennarichard/Hello-world/archive/refs/heads/main.zip), or run the following [git](https://git-scm.com/downloads) command to clone the files to your machine:
```

```bash
git clone git@github.com:ikennarichard/Hello-world.git
```

```
- Once the files are on your machine, open the _portfolio_ folder in [Visual Studio Code](https://code.visualstudio.com/)

- With the files open in Visual Studio Code, press the **Go Live** button at the bottom of the window to launch the files with [Live Server](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer).

### Install

The following tools help make easier to work with sample code.

- [git](https://git-scm.com/downloads): A tool for managing source code
- [Visual Studio Code](https://code.visualstudio.com/): A source code editor
- [Live Server](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer): A simple web server utility for Visual Studio Code

File: .github/workflows/linters.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
+name: Linters
```
Comment (topeogunleye) [inline .github/workflows/linters.yml:1]: >- [ ] Kindly ensure your linter results are running and that they pass * AKA: Linter should report NO errors as seen in the image below.

image

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @ikennarichard, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * Good commit messages. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._
- [ ] Please kindly add a descriptive PR title and description as seen in the demo below:
pull_request_desc
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -1 +1,161 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (binhussen) [inline README.md:159]: - [Optional] Great job for implementing previous code reviewer suggestions :+1: , I think your MIT link is broken, add the license and fix the broken link :+1: you can use [this template](https://github.com/microverseinc/readme-template/blob/master/MIT.md) and replace it with  your info

**Review by binhussen — APPROVED** [review]
Hi @ikennarichard , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**